api/v1/helpers/nouns/users.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
     POST: `Create a new ${m}`,
     PUT: `Overwrite all attributes of this ${m}`,
   },
+  scopeMap: {
+    withoutSensitiveInfo: 'withoutSensitiveInfo',
```

Comment:
I have a few tests to make sure that the password filed is not returned. Can I skip it for now?